heat/core/printing.py

```diff
@@ -0,0 +1,175 @@
+import copy
+import io
+import torch
+
+from typing import Dict
+
+__all__ = ["get_printoptions", "set_printoptions"]
+
+
+# set the default printing width to a 120
+_DEFAULT_LINEWIDTH = 120
+torch.set_printoptions(profile="default", linewidth=_DEFAULT_LINEWIDTH)
+
+
+def get_printoptions() -> Dict:
+    """
+    Returns the currently configured printing options.
+    """
+    return copy.copy(torch._tensor_str.PRINT_OPTS.__dict__)
+
+
+def set_printoptions(
+    precision: int = None,
+    threshold: int = None,
+    edgeitems: int = None,
+    linewidth: int = None,
+    profile: int = None,
+    sci_mode: int = None,
+):
```

Comment:
missing return value

Reply:
There is no return value on this function

Reply:
The python documentation uses `-> None`

Reply:
Fixed